tasks/src/image-classification/about.md

```diff
@@ -0,0 +1,26 @@
+## Use Cases
+Image classification is used for wide range of problems, from medical imaging to remote sensing. An example application is determining whether a Computed Tomography scan contains cancerous tissue or not.
+## Inference
+Inference
```

Comment:
nit: remove?

Reply:
which part?

Reply:
I see Inference both in a heading and then again